installer/volcano-development.yaml

```diff
@@ -121,7 +121,7 @@ spec:
       
       containers:
         - name: volcano-scheduler
-          image: volcanosh/vc-scheduler:latest
+          image: volcanosh/vc-scheduler:v0.3
```

Comment:
Hmm, i don't think we should do this. As master branch is the developing branch now, and this yaml is for developers, not for production use.

Reply:
we should make sure our release work as expected :)